packages/docs/src/styleguide.md

```diff
@@ -33,9 +33,41 @@ relevant sections in the sidebar for details about the styling & component APIs
   Most packages consist of JS and CSS resources, so please make sure you're including both in your application.
 </div>
 
+### TypeScript
+
+Blueprint is written in [TypeScript](https://www.typescriptlang.org/), a typed superset of
```

Comment:
"statically typed". It's important to be precise here; JS is already _dynamically_ typed, like every programming language